### PR TITLE
Feature/general bill

### PR DESCRIPTION
**Pull request by hlca**
tmp

### CODE REVIEW

File: src/Enum/General/GeneralAddressXML.php
Comment (hlca) [inline src/Enum/General/GeneralAddressXML.php:1]: Estos "case" lo normal es manejarlos con PascalCase

Address
PostalCode
... etc

File: src/Enum/General/AddressType.php
Comment (hlca) [inline src/Enum/General/AddressType.php:1]: Formatting

File: src/Documents/Generic/General/GeneralItems.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    use XMLWritterTrait;
+    public function __construct(
+        private Item $item,
+        private ProductServiceType $productServiceType,
+        // private TaxDetails $taxDetails
+    ) {}
+
+    public function asXML(): string
+    {
+        $sub_elements = [
```
Comment (hlca) [inline src/Documents/Generic/General/GeneralItems.php]: Naming convention

File: src/Documents/Generic/General/GeneralItems.php
Comment (hlca) [inline src/Documents/Generic/General/GeneralItems.php:1]: General formatting

File: src/Enum/General/GeneralCustomerXML.php [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?php 
+
+namespace SchoolAid\FEL\Enum\General;
+
+enum GeneralCustomerXML: string
+{
+    case Tag = 'dte:Receptor';
+    case IdCustomer = 'NITReceptor';
```
Comment (hlca) [inline src/Enum/General/GeneralCustomerXML.php]: Change IdCustomer with: TaxId

File: src/Models/Customer.php
Comment (hlca) [inline src/Models/Customer.php:1]: Change idCustomer with taxId

File: src/Models/Item.php
Comment (hlca) [inline src/Models/Item.php:1]: Use camel case when naming variables

File: src/Models/TaxDetails.php
Comment (hlca) [inline src/Models/TaxDetails.php:1]: Use camel case when naming variables

File: src/Traits/XMLWritterTrait.php [hunk 3]
```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace SchoolAid\FEL\Traits;
+
+trait XMLWritterTrait
```
Comment (hlca) [inline src/Traits/XMLWritterTrait.php]: Change this name to something like: "HasXML"

File: tests/Unit/GeneralDataTest.php
Comment (hlca) [inline tests/Unit/GeneralDataTest.php:1]: Missing assertions, use https://pestphp.com/docs/expectations#content-tomatchstring-expression to match the generated XML

File: src/Enum/General/GeneralAddressXML.php
Comment (uriel212) [reply src/Enum/General/GeneralAddressXML.php:1]: DONE

File: src/Enum/General/AddressType.php
Comment (uriel212) [reply src/Enum/General/AddressType.php:1]: Done

File: src/Documents/Generic/General/GeneralItems.php (hunk 1 above)
Comment (uriel212) [reply src/Documents/Generic/General/GeneralItems.php]: Done

File: src/Documents/Generic/General/GeneralItems.php
Comment (uriel212) [reply src/Documents/Generic/General/GeneralItems.php:1]: Done

File: src/Enum/General/GeneralCustomerXML.php (hunk 2 above)
Comment (uriel212) [reply src/Enum/General/GeneralCustomerXML.php]: Done

File: src/Models/Item.php
Comment (uriel212) [reply src/Models/Item.php:1]: Applied for all the models created

**Pull request merged**